src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color, scale $transition-duration;
```

Comment:
```suggestion
    transition: color $transition-duration, scale $transition-duration;
```

Reply:
if you need to apply this time to both properties you need to write it after each one